core/src/main/java/nl/nn/adapterframework/jdbc/migration/LiquibaseLogger.java

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2021 WeAreFrank!
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package nl.nn.adapterframework.jdbc.migration;
+
+import liquibase.changelog.ChangeSet;
+import liquibase.changelog.DatabaseChangeLog;
+import liquibase.logging.core.AbstractLogger;
+import nl.nn.adapterframework.util.LogUtil;
+
+import org.apache.logging.log4j.Logger;
+
+/**
+ * @author alisihab
+ *
+ */
+public class LiquibaseLogger extends AbstractLogger {
+	private static final Logger log = LogUtil.getLogger(LiquibaseLogger.class);
+
+	@Override
+	public void setName(String name) {
+	}
```

Comment:
Give a brief explanation that calling this wont do much as the loggers name has already been defined upon creating the logger